definitions/npm/eventemitter3_v2.x.x/flow_v0.25.x-/eventemitter3_v2.x.x.js

```diff
@@ -1,5 +1,5 @@
 declare module 'eventemitter3' {
-  declare type ListenerFn = (...args: any[]) => void
+  declare type ListenerFn = (...args: any[]) => mixed
```

Comment:
Could it be `void | Promise<*>` instead of `mixed?

Reply:
As I learned recently `() => void` is equal `() => mixed` in flow.js:
[Try TS () => void](https://www.typescriptlang.org/play/#src=declare%20function%20AA(fn%3A%20(...args%3A%20Array%3Cany%3E)%20%3D%3E%20void)%3A%20void%3B%0D%0A%0D%0AAA(async%20()%20%3D%3E%20%7B%20%7D)%3B%0D%0AAA(async%20()%20%3D%3E%202)%3B%0D%0AAA(()%20%3D%3E%20%7B%20%7D)%3B%0D%0AAA(()%20%3D%3E%201)%3B%0D%0AAA(()%20%3D%3E%20'')%3B%0D%0AAA(()%20%3D%3E%20%5B1%5D)%3B)


Based on official ts defenitions: https://github.com/primus/eventemitter3/blob/bb9242b0f78f04d8faae9ae06623c8003ffeedb8/index.d.ts#L54

Reply:
They reverted the change...